summarization/text_entity_detection.py

```diff
@@ -0,0 +1,45 @@
+import base64
+import json
+import os
+
+import requests
+
+from summarization.bad_request_error import BadRequestError
+
+
+class TextEntityRetriever:
+
+    API_ENDPOINT \
+        = "https://language.googleapis.com/v1beta2/documents:analyzeEntities?key="  # noqa
```

Comment:
Try avoiding the use of #noqa whenever possible. It is generally not a good practice to disable linters on specific lines.

Reply:
+1